dpnp/backend/kernels/dpnp_krnl_arraycreation.cpp

```diff
@@ -252,8 +240,8 @@ void dpnp_trace_c(const void* array1_in, void* result1, const size_t* shape_, co
     auto memcpy_event = DPNP_QUEUE.memcpy(shape, shape_, ndim * sizeof(size_t));
 
     cl::sycl::range<1> gws(size);
-    auto kernel_parallel_for_func = [=](cl::sycl::id<1> global_id) {
-        size_t i = global_id[0];
+    auto kernel_parallel_for_func = [=](auto index) {
```

Comment:
removing any explicit type definition from any code - makes this code less understandable and scalable.